### DISCUSSION TITLE
ED25519 for Java 11

### DISCUSSION THREAD

**Discussion by gavinbaumanis** [bcgit/bc-java#1547] [Q&A] [answered]
Hi There I am using an Application server ([Lucee](https://www.lucee.org) - that compiles Coldfusion Markup Language (CFML) to Java bytecode.

It comes with bouncy castle (mail,prov and tsp) at version 1.38
![image](https://github.com/bcgit/bc-java/assets/14960023/31ace869-8ae4-4d25-94ed-8dd594d69f39)

When attempting to use CFML's built in functions for operating with SFTP servers, I am unable to use ED25519 flavoured certifcates for authentication. After some google-fu exercising I "believe" I need to use Java 17 or manually use an appropriate version of BC jars that support ED25519.

The Lucee server allows for extra Jars to be auto-loaded into the running application server.
Ultimately it runs a servlet container upon Apache Tomcat(v9.latest) and uses Java 11.(latest)

Can I please ask for some help, with respect to what versions of the BC libraries I would need to enable ED25519 handling?

Thanks!

**Comment by dghgit** [comment, marked as answer]
This answer is way late, but my best recommendation would be to get the container moved to the LTS release. We're at 1.78 now, 1.38 is almost in the "do not use" class now, a lot has changed since that came out!
